<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pw_dialog</class>
 <widget class="QDialog" name="pw_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Old Password</string>
           </property>
           <property name="buddy">
            <cstring>old_pw_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>New password</string>
           </property>
           <property name="buddy">
            <cstring>new_pw_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>New password again</string>
           </property>
           <property name="buddy">
            <cstring>confirm_edit</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLineEdit" name="old_pw_edit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="new_pw_edit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="confirm_edit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>WARNING: for security reasons, Dwyco will not fetch lost passwords. It is strongly recommended that you enter an email (in your profile) you can  use to reset your password.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="email_label">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;Privacy note: your email address is never disclosed to third parties.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="fetch_pw_button">
       <property name="text">
        <string>Fetch Lost Password...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="save_pw_checkbox">
     <property name="text">
      <string>Check this box to start CDC-X without a password.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>old_pw_edit</tabstop>
  <tabstop>new_pw_edit</tabstop>
  <tabstop>confirm_edit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pw_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pw_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_pw_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>new_pw_edit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_pw_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>confirm_edit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_pw_checkbox</sender>
   <signal>clicked()</signal>
   <receiver>new_pw_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_pw_checkbox</sender>
   <signal>clicked()</signal>
   <receiver>confirm_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_pw_checkbox</sender>
   <signal>clicked()</signal>
   <receiver>old_pw_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_pw_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>old_pw_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
